.github/workflows/conformance-eks-v1.11.yaml

```diff
@@ -0,0 +1,345 @@
+name: ConformanceEKS (ci-eks-1.11)
```

Comment:
I think we can actually just drop this PR, if this is introducing the whole workflow then it's probably not necessary on the v1.11 branch. My bad.

Reply:
It got rid of something as well.

Reply:
I only see new files in https://github.com/cilium/cilium/pull/18109/commits/ba974dc72303a1909adda0c33014cde48ffc96b5 .

Reply:
NVM. Removed.